src/components/playback/playbackmanager.js

```diff
@@ -145,7 +145,10 @@ function createStreamInfoFromUrlItem(item) {
 }
 
 function mergePlaybackQueries(obj1, obj2) {
-    const query = Object.assign(obj1, obj2);
+    const query = obj1;
+    for (const key in obj2) {
+        if (obj2[key] !== undefined) query[key] = obj2[key];
```

Comment:
Maybe just use the lodash merge implementation?

Reply:
@v0idMrK what do you think about using lodash for this?

Reply:
@thornbill 

Hey, sorry for only answering now! Been super busy lately.

It sounds good to me. I will see if today Im able to change it quickly.

Thank you for the heads up!